src/program/lwaftr/tests/selftest.sh

```diff
@@ -6,18 +6,18 @@
 export TESTS_DIR=`dirname "$0"`
 export PYTHONPATH=${TESTS_DIR}
 
-# Only run tests in the passed subdirectory of $TESTS_DIR.
+# Only run tests in the chosen test file (without the _test.py suffix).
 if [[ -n $1 ]]; then
```

Comment:
Right. I assumed we weren't going to have spaces in test file names, but it doesn't hurt to be defensive anyway. I'll change it.